src/app/loans/loans-view/transactions/view-transaction/view-transaction.component.html

```diff
@@ -18,91 +23,139 @@
     <mat-card-content>
 
       <div fxLayout="row wrap" class="content">
+        <div fxFlex="100%" [ngClass]="loanTransactionColor()">
+        </div>
 
         <div fxFlex="50%" class="mat-body-strong">
-          Transaction Id
+          {{ 'Transaction Id' | translate }}
         </div>
 
         <div fxFlex="50%">
           {{ transactionData.id }}
         </div>
 
+        <div fxFlex="50%" class="mat-body-strong" *ngIf="transactionData.toLoanTransactionRelation">
+          {{ 'Repayment Transaction' | translate }}
+        </div>
+
+        <div fxFlex="50%" *ngIf="transactionData.toLoanTransactionRelation">
+          <a [href]="loanTransactionRelatedLink(transactionData.toLoanTransactionRelation.fromLoanTransaction)">
+              {{ transactionData.toLoanTransactionRelation.fromLoanTransaction }}
+          </a>
+        </div>
+
         <div fxFlex="50%" class="mat-body-strong">
-          Type
+          {{ 'Type' | translate }}
         </div>
 
         <div fxFlex="50%">
           {{ transactionData.type.value }}
         </div>
 
         <div fxFlex="50%" class="mat-body-strong">
-          Transaction Date
+          {{ 'Transaction Date' | translate }}
```

Comment:
Transaction date for chargeback is not allowed. It will be always on the actual business date

Reply:
Done!